src/main/java/org/sagebionetworks/bridge/udd/dynamodb/DynamoHelper.java

```diff
@@ -1,24 +1,28 @@
 package org.sagebionetworks.bridge.udd.dynamodb;
 
+import java.io.IOException;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import javax.annotation.Resource;
 
 import com.amazonaws.services.dynamodbv2.document.Index;
 import com.amazonaws.services.dynamodbv2.document.Item;
 import com.amazonaws.services.dynamodbv2.document.RangeKeyCondition;
 import com.amazonaws.services.dynamodbv2.document.Table;
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Multimap;
 import org.springframework.stereotype.Component;
 
-import org.sagebionetworks.bridge.udd.accounts.AccountInfo;
-import org.sagebionetworks.bridge.udd.worker.BridgeUddRequest;
-
 /** Helper class to wrap some Dynamo DB queries we make. */
 @Component
 public class DynamoHelper {
     private Table ddbHealthIdTable;
     private Table ddbStudyTable;
-    private Index ddbUploadTableIndex;
+    private Table ddbSynapseMapTable;
+    private Table ddbUploadSchemaTable;
+    private Index ddbUploadSchemaStudyIndex;
```

Comment:
Because typing ddb instead of dynamo saves me 3 keystrokes d=

On a more serious note the Dynamo[Name] pattern in BridgePF refers to the classes themselves and use the Mapper API, while these vars use the Table API.

Unless you mean the Mapper vs Table API and not just the names. In which case the real answer is I didn't want to copy-paste over a whole bunch of DDB code from BridgePF. There's a JIRA tracking consolidating this https://sagebionetworks.jira.com/browse/BRIDGE-763